src/countryflag/countryflag.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import countryflag from 'countryflag'
+
+class CountryFlag extends Component {
+  render() {
+    const { iso2, svg } = this.props
+    const flag = countryflag(iso2)
+    return svg === true || flag.emoji === null ? (
+      <img style={{ width: '18px' }} alt={flag.name} src={flag.svg} />
+    ) : (
+      <span>{flag.emoji}</span>
```

Comment:
Returning a span can create unwanted effects, is it necessary?

Reply:
I agree on having a span if we add this accessibility aspect: https://tink.uk/accessible-emoji/